source/mainnet/smart-contracts/tutorials/piggy-bank/deploying.rst

```diff
@@ -0,0 +1,114 @@
+.. code-block:: console
+
+   $cargo concordium build
+
+.. Note::
+   This command creates a module ``piggy_bank_part2.wasm.v1`` in the folder ``./target/concordium/wasm32-unknown-unknown/release/``. Please use the wasm module file suffixed with `v1` which is the newest smart contract version supported by the Concordium chain.
+
+The ``piggy_bank_part2.wasm.v1`` file can be deployed to the Concordium chain. Let’s transfer the ``piggy_bank_part2.wasm.v1`` file to the place where you are running your concordium-client tool. Move to the folder as well.
+
+Deploying a module
+==================
+You are now set up to deploy the piggy bank module to the testnet chain.
+
+.. Note::
+   Please remember to adjust the following commands when you are running your ``concordium-client`` tool locally and want to connect to your node on the server. You can look up the different options on how to connect the ``concordium-client`` tool to your testnet node :ref:`here<interacting_with_your_testnet_node>`.
+
+.. code-block:: console
+
+   $./concordium-client module deploy piggy_bank_part2.wasm.v1 --sender <Account name> --grpc-port 10001
+
+
+.. Note::
+   This tutorial uses ``Concordium`` for the <Account name>.
+
+.. Note::
+   You will be asked to input a password. It is the password that you use for your mobile app wallet log-in.
+
+If everything works correctly the output has a green line with your module reference. Remember this module reference since it will be used in the next step. You can find additional information about deploying a module in this :ref:`guide<deploy-module>`.
+
+Initializing a module
+=====================
+
+You can initialize your smart contract now to create a smart contract instance.
+
+.. code-block:: console
+
+   $./concordium-client contract init <module reference> --sender <Account name> --contract PiggyBank --energy 1000 --grpc-port 10001
+
+.. Note::
+   Use enough energy to initialize the piggy bank smart contract.
+
+
+If everything works correctly the output has a green line with your smart contract instance index. Remember this index since it will be used in the next step.
+
+.. code-block:: console
+
+   Contract successfully initialized with address: {"index":<Smart contract instance index>,"subindex":0}
+
+You can find additional information about initializing a smart contract instance in this :ref:`guide<initialize-contract>`.
+
+
+Updating the piggy bank smart contract
+======================================
+
+Let's deposit 1 CCD into the piggy bank to test if we can update the smart contract instance.
+
+.. code-block:: console
+
+   $./concordium-client contract update <Smart contract instance index> --entrypoint insert --energy 1000 --sender <Account name> --amount 1 --grpc-port 10001
+
+If everything works correctly the output has a green line as follows:
+
+.. code-block:: console
+
+   Successfully updated contract instance {"index":<Smart contract instance index>,"subindex":0} using the function 'insert'.
```

Comment:
```suggestion
   Successfully updated contract instance {"index":<Smart contract instance index>,"subindex":0}(piggy_bank_part2) using the function 'insert'.
```
I believe this is how contracts with names are formatted. But please check to make sure.